### PR TITLE
Fix: add articles metadata

### PR DESCRIPTION
**Pull request by tejaswinichile**
fixes: https://github.com/chatwoot/product/issues/497

### CODE REVIEW

File: app/views/api/v1/accounts/portals/_portal.json.jbuilder [hunk 1]
```diff
@@ -17,3 +17,9 @@ json.portal_members do
     end
   end
 end
+
+json.meta do
+  json.articles_count portal.articles.published.size
```
Comment (muhsin-k) [inline app/views/api/v1/accounts/portals/_portal.json.jbuilder:22]: Shouldn't we return all the articles?

Comment (muhsin-k) [reply app/views/api/v1/accounts/portals/_portal.json.jbuilder:22]: Screenshot 2022-07-27 at 3 00 01 PM

Comment (tejaswinichile) [reply app/views/api/v1/accounts/portals/_portal.json.jbuilder:22]: It was mentioned in one of the tickets about the published articles count.

**Comment by muhsin-k** [comment]
I have found 2 issues while testing the API. Please check this video

https://www.loom.com/share/030081b0e7f34dc08ee9c095140cc3a5

**Comment by tejaswinichile** [comment]
@muhsin-k I have changed the filter params in the `payload` key, but removing that since we are not doing that in any other filters.

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**